contributing.md

```diff
@@ -1,5 +1,5 @@
 
-## How to add and make changes to the curriculum
+## How to add and make changes to the curriculum ??
```

Comment:
I do not think we need a question mark here. Especially not two. This is a title of a section, not a question to be asked